admin_guide/configuring_authentication.adoc

```diff
@@ -69,6 +69,12 @@ challenges should set this header to a non-empty value.
 Not supported by all identity providers.
 |===
 
+[NOTE]
+When changing authentication identity providers, ensure the `*name*` setting
```

Comment:
They should only use the same IdP name if the new provider is actually identifying the same set of users (for example switching from an auth proxy integration to an LDAP integration, but with the same backing users)